spotify

```diff
@@ -188,8 +188,7 @@ while [ $# -gt 0 ]; do
 
         "vol"    )
             vol=`osascript -e 'tell application "Spotify" to sound volume as integer'`;
-            text=$2;
-            if [[ "$2" = "show" || "$2" = "" ]]; then
```

Comment:
Note that the old behaviour wasn't so bad. In that we could perhaps just change the docs to not show `vol show` and instead just show `vol`. Apparently it would work as you want.